pkg/types/block.go

```diff
@@ -43,40 +43,38 @@ type BlockRecord struct {
 
 // FullBlock a full block
 // https://github.com/Chia-Network/chia-blockchain/blob/0befdec071f49708e26c7638656874492c52600a/chia/types/full_block.py#L16
```

Comment:
```suggestion
// https://github.com/Chia-Network/chia_rs/blob/main/crates/chia-protocol/src/fullblock.rs#L13
```

Just nice to keep the canonical source up to date. This recently moved to rust.